internal/provider/schema_cloudflare_notification_policy.go

```diff
@@ -155,7 +249,79 @@ func notificationPolicyFilterSchema() *schema.Schema {
 						Type: schema.TypeString,
 					},
 					Optional:    true,
-					Description: "A numerical limit. Example: `99.9`",
+					Description: "A numerical limit. Example: `99.9`.",
+				},
+				"requests_per_second": {
```

Comment:
you'll need to reference these in the resource code itself for it to do anything with the API (see https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/resource_cloudflare_notification_policy.go).

Reply:
The changes to the API I made are to include missing types to the schema that is already referenced in https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/resource_cloudflare_notification_policy.go#L17 so that the validation(https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/schema_cloudflare_notification_policy.go#L35) doesn't error when a policy is configured for the missing type. Please let me know if I am missing something.

Reply:
sorry, i probably wasn't clear.

just adding these fields to the schema doesn't mean they end up in the API requests. they will need to be added to the functionality (potentially, https://github.com/cloudflare/terraform-provider-cloudflare/blob/b55dc6891b109e090f608a0e05e7661a07be4429/internal/provider/resource_cloudflare_notification_policy.go#L135) for them to be added to the API interactions, persist to state and have an effect on the resources. does that help?

Reply:
Thanks for explaining. Sorry about that. I added those fields to just update documentation for now. We have updated one of our APIs (available alerts) to return filter information so that we don't have to update this doc regularly. I will work on adding that to terraform soon where we can control what filters can be defined per alert type. Returning and accepting filters as a map should be fine for now. Upstream will return an error if the filter fields are not as expected.